# HG changeset patch
# User Rik <rik@octave.org>
# Date 1442912777 25200
#      Tue Sep 22 02:06:17 2015 -0700
# Branch stable
# Node ID 1339ad3c9faa2fb7cdb5e2c9d1de89ee99682597
# Parent  635c93bba6b606090c40a9ad6e9304878a30c1a1
Fix typo in str2func docstring.

* ov-fcn-handle.cc (Ffunc2str): Fix typo where 'a' should be '@'.

diff --git a/libinterp/octave-value/ov-fcn-handle.cc b/libinterp/octave-value/ov-fcn-handle.cc
--- a/libinterp/octave-value/ov-fcn-handle.cc
+++ b/libinterp/octave-value/ov-fcn-handle.cc
@@ -1851,18 +1851,18 @@ DEFUN (str2func, args, ,
 @deftypefn  {Built-in Function} {} str2func (@var{fcn_name})\n\
 @deftypefnx {Built-in Function} {} str2func (@var{fcn_name}, \"global\")\n\
 Return a function handle constructed from the string @var{fcn_name}.\n\
 \n\
 If the optional @qcode{\"global\"} argument is passed, locally visible\n\
 functions are ignored in the lookup.\n\
 \n\
 Note: @code{str2func} does not currently accept strings which define\n\
-anonymous functions (those which begin with @samp{a}).\n\
-Use @code{eval (@var{str})} as a replacement.\n\
+anonymous functions (those which begin with @samp{@@}).\n\
+Use @w{@code{eval (@var{str})}} as a replacement.\n\
 @seealso{func2str, inline}\n\
 @end deftypefn")
 {
   octave_value retval;
   int nargin = args.length ();
 
   if (nargin == 1 || nargin == 2)
     {
